cumulus_library_opioid/manifest.toml

```diff
@@ -5,6 +5,7 @@ file_names = [
     "vocab/static_builder.py",
     "vocab/rxnorm_vsac_builder.py",
     "vocab/additional_rules_builder.py",
+    "vocab/excluded_rules_summary.py",
```

Comment:
that's the wrong filename, right?  I think you want `excluded_rules_builder.py`

Reply:
changed it everywhere but here, thanks